context_prop/src/main/java/io/opentelemetry/context/propagation/TextMapPropagator.java

```diff
@@ -115,6 +116,14 @@
    */
   interface Getter<C> {
 
+    /**
+     * Returns all the keys in the given carrier.
+     *
+     * @param carrier carrier of propagation fields, such as an http request.
+     * @since 0.10.0
```

Comment:
I think we've gotten rid of all the `@since` tags for now; we'll set them all to 1.0 when we're ready to release.

Reply:
Oh, I totally forgot about it. Will do.